recipes/syscalls-cpp/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.15)
+project(test_package CXX)
+
+find_package(syscalls-cpp REQUIRED CONFIG)
+
+add_executable(test_package src/main.cpp)
+
+target_link_libraries(test_package PRIVATE syscalls-cpp::syscalls-cpp)
```

```suggestion
add_executable(test_package main.cpp)
target_link_libraries(test_package PRIVATE sapdragon::syscalls-cpp)
```

- Fix incorrect source file path
- Follow the upstream CMake target name